### ISSUE TITLE
Add lang-Attribute note

### ISSUE THREAD

**Issue by MarcoHengstenberg** [thedaviddias/Front-End-Checklist#86]
It would make a lot of sense to tell developers that a lang-Attribute is always needed in the HTML element (first section with doctype).
The point is that screenreaders will automatically switch their language-mode when a lang attribute is present. That way the reader will not announce in a different language than present on the page.

**Comment by thedaviddias** [comment]
Hi @MarcoHengstenberg, I'm not sure if I clearly understand your request... Did you saw that the `lang`is in the [next section](https://github.com/thedaviddias/Front-End-Checklist#html-tags)?

**Comment by MarcoHengstenberg** [comment]
Oh. Nope, I didn't see that. After not seeing the lang-Attribute on the `<html>` element I search the readme for "lang-attribute" to no avail.
Then I'd suggest renaming that section to `lang-Attribute` instead of `Language tag`. :)

**Comment by thedaviddias** [comment]
Voilà :)

**Comment by MarcoHengstenberg** [comment]
Merci Monsieur 😁

**Comment by MarcoHengstenberg** [comment]
> Language attribute: High The language tag of your website is specified and related to the language of the current page.

Oh noes.

**Comment by thedaviddias** [comment]
@MarcoHengstenberg better now?

**Comment by MarcoHengstenberg** [comment]
Wonderful! Many thanks. :)
